README.md

```diff
@@ -10,25 +10,25 @@ We have implemented connection pooling and other interesting features.
 
 ### Denylist Query Rejection
 
-MaxScale uses an embedded MySQL parser for query routing. It builds a parse tree for every MySQL query in its query classifier module. Airbnb MaxScale leverages the query parse tree for bad query blacklisting. The motivation for this feature was a Ruby VM heap memory corruption in our production environment. Because of the memory corruption, MySQL query statement generated by Rails ActiveRecord could be corrupted in such way that its conditional predicate was completely removed. For instance, one corrupted query that we were lucky to catch before it making permanent damage in production was a delete statement with `where 0 = 0`. This had put our production databases in the danger of humongous corruption. This [blog post](http://webuild.envato.com/blog/tracking-down-ruby-heap-corruption/) has a detailed explanation of the nasty Ruby heap corruption problem. Airbnb MaxScale looks for existence of malformed equality conditions in update and delete statements by inspecting the predicate in the query parse tree, and reject them flatly.
+MaxScale uses an embedded MySQL parser for query routing. It builds a parse tree for every MySQL query in its query classifier module. Airbnb MaxScale leverages the query parse tree for bad query blacklisting. The motivation to build this feature came from a Ruby VM heap memory corruption incident in our production environment. Because of the memory corruption, Rails' ActiveRecord generated a MySQL query statement was corrupted in such way that its conditional predicate was completely removed. For instance, one corrupted query that we were lucky to catch before it caused permanent damage in production was a delete statement with `where 0 = 0`. This had put our production databases in danger of serious corruption. This [blog post](http://webuild.envato.com/blog/tracking-down-ruby-heap-corruption/) has a detailed explanation of the nasty Ruby heap corruption problem. Airbnb MaxScale looks for existence of malformed equality conditions in update and delete statements by inspecting the predicate in the query parse tree and rejecting them.
 
 ### Monitoring
 
 Airbnb MaxScale implements minutely internal metrics collection and exposes them for real time monitoring. A new `stats` command is added in MaxAdmin tool. External stats agent can be created to pull stats from a running MaxScale server periodically.
```

Comment:
@liangg I didn't understand what "minutely internal metrics collection" meant. Can you clarify that part?